src/test/java/task30/ByVariables.java

```diff
@@ -0,0 +1,11 @@
+package task30;
+
+public class ByVariables {
+    public static String SignInXPath = "/html/body/div[2]/header/div[1]/div/ul/li[2]/a";
+    public static String EmailInputXPath = "//*[@id=\"email\"]";
+    public static String PasswordInputXPath = "//*[@id=\"pass\"]";
+    public static String SignInButtonXPath = "//*[@id=\"send2\"]/span";
+
+    public static String AssertXPath = "/html/body/div[2]/header/div[1]/div/ul/li[1]/span";
```

Comment:
1. do not use auto-generated x-paths, in case of UI update those locators will cause test failure first
2. "//*[@id=\"email\"]" could be updated to "//*[@id='email']"
3. let's use By type to represent locators instead of string